### ISSUE TITLE
Feature request: Object detection model zoo

### ISSUE THREAD

**Issue by austinmw** [pytorch/pytorch#8090]
Are there any plans for an object detection model zoo like tensorflow has, with models like ssd, faster r-cnn, mask r-cnn, etc? I really like pytorch but am currently using tensorflow to avoid using unofficial/nonsupported model repos. An official object detection model zoo along with a standardized data format for bounding box/segmentation annotations would be amazing.

**Comment by vadimkantorov** [comment]
I wish there was a **contrib** repo for Models with relaxed conditions on accepting PRs (e.g. with little or no tests) with self-contained Module classes, but instead that would allow for tracking usage (through an opt-in), that way the PyTorch team can have information on popular models and focus on bringing them in a better shape. If it's refactored and a new version is available, a warning message can be shown to users. I'm thinking of a very-very lightweight package manager within PyTorch that would let importing a model from the web dynamically like `from torch.contrib import roi_pooling`, and that would download a model and place it to a local folder.

An advantage over using other people's repos is predictability/easeness of importing (we know that no obscure build steps or dependencies are needed), and some guarantee that the code will stay available for some time. Then some popular models can be turned into fully-blown examples that can be used for testing PyTorch performance.

I understand, there are a lot of design/ecosystem/maintenance considerations here, but it would allow to catch up with `tensorflow.contrib`.

**Comment by ezyang** [comment]
cc @colesbury

**Comment by fmassa** [comment]
Torchvision now has models for object detection and semantic segmentation officially supported
